### PR TITLE
CPCCICD-1045 Enable log forwarding

### PR DESCRIPTION
**Pull request by holgpar**
### Description

We want to add support for log-forwarders (i.e. fluentd) instead of sending logs directly to OpenSearch/ElasticSearch. For this purpose, two configuration extension for the Helm chart are prepared:

* Set environment variables to configure the elasticsearch-log-plugin to forward data to fluentd
* Enable the use of tekton sidecars. This can be used to run the forwarder as a sidecar container in the JFR pod

Note: With the current version of the JFR image, the plugin fails at startup when configured to use fluentd. This is addressed by this fix: https://github.com/SAP/stewardci-jenkinsfilerunner-image/pull/101

### Submitter checklist

- [x] Change has been tested (on a back-end cluster)
- [x] (If applicable) Jira backlog item ID added to the PR title and the [changelog.yaml] entry
- [x] [changelog.yaml] entry with upgrade notes is prepared and appropriate for the audience affected by the change (users or developer, depending on the change)
- [x] Semantic version diffed against [last release][releases] and updated accordingly. In this project the version has to be maintained here:
    - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) (`version` and `appVersion`)

In case dependencies have been updated:
- [ ] Links to external changelogs, since the last release of our component, added to the [changelog.yaml] entry (description).
- [ ] Changelogs read thoroughly, potential impact described, upgrade notes prepared (if necessary)
- [ ] Check if dependency updates affect our semantic version increment.

### Reviewer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There is at least 1 approval for the pull request and no outstanding requests for change
- [ ] All voter checks have passed
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] The Pull Request title is understandable and reflects the changes well
- [ ] The Pull Request description is understandable and well documented
- [ ] [changelog.yaml] entry for this Pull Request has been added
    - [ ] Changelog entry contains all required information
    - [ ] 'Upgrade notes' are documented in changelog.yaml (if required)

[changelog.yaml]: https://github.com/SAP/stewardci-core/changelog.yaml
[releases]: https://github.com/SAP/stewardci-core/releases

### CODE REVIEW

**Review by LenaHoinkis — APPROVED** [review]
lgtm. 
For me personally the comments {{/* if */}} behind the end statements is a bit confusing.

**Comment by holgpar** [comment]
> For me personally the comments {{/* if */}} behind the end statements is a bit confusing.

I will remove them.

File: charts/steward/values.yaml [hunk 1]
```diff
@@ -84,6 +84,18 @@ pipelineRuns:
...
+      enabled: false
+      # The hostname of the forwarder
+      host: ""
+      # If "true", the fowarder is expected to run as a sidecare of the jenkinsfileRunner pod.
+      # In that case "host" will be ignored.
+      useSidecar: true
+      # The port the forwarder is listening to
+      port: '24224'
+      # The tag to use when sending data to the forwarder
+      tag: BUILD_LOGS
```
Comment (anbrsap) [inline charts/steward/values.yaml:98]: Please add the documentation to the chart README.

Comment (holgpar) [reply charts/steward/values.yaml]: Done.

**Pull request merged**